service/src/main/kotlin/no/nav/su/se/bakover/service/utbetaling/UtbetalingServiceImpl.kt

```diff
@@ -64,6 +68,19 @@ internal class UtbetalingServiceImpl(
             } ?: FantIkkeUtbetaling.left()
     }
 
+    override fun hentGjeldendeUtbetaling(sakId: UUID, forDato: LocalDate): Either<FantIkkeGjeldendeUtbetaling, Utbetalingslinje> {
+        val utbetalingslinjer = hentUtbetalinger(sakId).flatMap { it.utbetalingslinjer }.filterIsInstance<Utbetalingslinje.Ny>()
```

Comment:
Hm, kan hende vi må tenkte litt på hvordan dette vil utarte seg for andre typer linjer (p.t opphør - kanskje også hvil og reaktivering). I den konteksten kan det kanskje være litt uheldig at en såpass generell funksjon bare bryr seg om `Ny`.

Reply:
Ja, vi diskuterte dette en del og kom iallfall til at dette er det som virker mest logisk sånn ting er nå. Har du eventuelt noen forslag til hvordan det går an å gjøre dette mer future-proof?

Reply:
Ja, basert på funksjonsnavnet burde vi kanskje ikke filtrert på .Ny

Reply:
Ser den, men hvis vi ser på den casen som er forsøkt løst i denne PR-en så blir jo spørsmålet om det da blir riktig. For da kan det vel bli mulig at vi endre opp med et brev der det står `du vil fortsatt få 0 kroner` (gitt at det er det beløpet som ligger i opphørslinjer. Men kanskje det faktisk hadde vært mer riktig i det tilfellet?